wampy/transports/websocket/frames.py

```diff
@@ -263,7 +263,7 @@ def __init__(self, bytes):
 
         self.opcode = bytes[0] & 0b1111
 
-        if self.opcode != 9:
+        if self.opcode != 9 and self.opcode != 8:
```

Comment:
could you do
```
if self.opcode == 8:
    return
```
?
otherwise i think it's nicer to do
```
if self.opcode not in [8, 9]
```
either way i think it's worth adding a comment that the opcode 9 is the websocket close operation code.

thanks

Reply:
The CLOSE however appears to have an error code attached (2 bytes). I havent figured out how to dig it out, tho. So it would be probably better to have a dedicated message class for the CLOSE message and properly handled

Reply:
I'm not sure. wampy only has message classes for WAMP messages, and there is not CLOSE WAMP message. I hadn't really considered websocket messages until wampy had to support PING and CLOSE... but maybe I should. for now, i think what you've done is fine, and I'll think over explicitly handling websocket messages and whether it's necessary.

Reply:
Hey; I just found today there is another piece missing. I will commit it later today / tomorrow in this pr.